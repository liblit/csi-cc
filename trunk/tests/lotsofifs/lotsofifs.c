#include <stdio.h>

int foo(){
  return 3;
}

int main(){
  int x = 0, y = 10;
  int z = foo();
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y){
    if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  }
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y){
    if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  }
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  if(x < y)
    x++;
  x += foo();
  printf("%d\n", x+z);
}
