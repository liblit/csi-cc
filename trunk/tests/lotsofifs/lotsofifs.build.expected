WARNING: instrumentation not done for function main due to large path count.  Path info will be missing!
